src/main/java/dev/enjarai/trickster/datagen/ModBlockTagGenerator.java

```diff
@@ -20,6 +20,8 @@ public ModBlockTagGenerator(FabricDataOutput output, CompletableFuture<RegistryW
     protected void configure(RegistryWrapper.WrapperLookup wrapperLookup) {
         registerHeatConversion(Blocks.STONE, Blocks.DEEPSLATE);
         registerHeatConversion(Blocks.COBBLESTONE, Blocks.COBBLED_DEEPSLATE);
+        registerHeatConversion(Blocks.MOSSY_COBBLESTONE, Blocks.COBBLESTONE);
+        registerHeatConversion(Blocks.MOSSY_COBBLESTONE, Blocks.COBBLED_DEEPSLATE);
```

Comment:
Wait why are you registering mossy cobble twice?

Reply:
50% chance of one of either cobblestone or cobbled deepslate

Reply:
If that's what you want, use the varargs in one method call. I'm pretty sure this will overwrite the existing tag.

Reply:
The erosion trick was based on top of the heating one, it works fine

Reply:
I plan to rewrite this system in the future with a custom data loader with more features, it also would fix the warcrimes against tags that the mod has

Reply:
@enjarai, I do not believe we need to document all conversions. I think having them be something to experiment with could be interesting.

Reply:
I've seen this work on stream enjarai, doubled declarations don't appear to overwrite.

Reply:
Alright, nvm then.